src/main/java/com/myfirstownapp/beeapp/harvest/HarvestController.kt

```diff
@@ -0,0 +1,22 @@
+package com.myfirstownapp.beeapp.harvest
+
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/beeapp/harvest")
+class HarvestController (private val service:HarvestDbService){
+
+    @GetMapping(value = ["getHarvest"])
```

Comment:
`@GetMapping(all)` but I think it could be simple `@GetMapping` to get all data

Reply:
`@GetMapping` is working to get all data